### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by EmrysRoberts**
This PR adds integration tests for PUT and GET calls to corporate and natural endpoints.

**Resolves:**
- DSND-715

### CODE REVIEW

File: src/itest/java/uk/gov/companieshouse/disqualifiedofficersdataapi/steps/CorporateDisqualificationSteps.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    @Autowired
+    private DisqualifiedOfficerRepository repository;
+
+    @Autowired
+    private MongoTemplate mongoTemplate;
+
+    @Autowired
+    public DisqualifiedOfficerApiService disqualifiedApiService;
+
+    @Before
```
Comment (amartin7663) [inline src/itest/java/uk/gov/companieshouse/disqualifiedofficersdataapi/steps/CorporateDisqualificationSteps.java:55]: Would you not also need an `@After` to stop the mongo container?

File: src/itest/resources/features/corporate_disqualification.feature [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  Scenario Outline: Processing corporate disqualified officer information successfully
+
+    Given disqualified officers data api service is running
+    When I send corporate PUT request with payload "<data>" file
+    Then I should receive 200 status code
+    And the CHS Kafka API is invoked successfully with "<officerId>"
+
+    Examples:
+      | data                                  | officerId     |
+      | corporate_disqualified_officer        | 1234567891    |
```
Comment (amartin7663) [inline src/itest/resources/features/corporate_disqualification.feature:12]: Is using a scenario outline with example necessary if you only have one example case. You could do this instead?
```suggestion
  Scenario: Processing corporate disqualified officer information successfully

    Given disqualified officers data api service is running
    When I send corporate PUT request with payload corporate_disqualified_officer file
    Then I should receive 200 status code
    And the CHS Kafka API is invoked successfully with 1234567891
```

Comment (EmrysRoberts) [reply src/itest/resources/features/corporate_disqualification.feature:12]: I think it's best to keep these files generic to make them easier to add to in future even if it means having a single example per scenario for the time being.

Comment (amartin7663) [reply src/itest/resources/features/corporate_disqualification.feature:12]: Makes sense

**Review by amartin7663 — APPROVED** [review]

**Pull request merged**